linode_api4/objects/serializable.py

```diff
@@ -141,3 +142,22 @@ def __delitem__(self, key):
 
     def __len__(self):
         return len(vars(self))
+
+
+class StrEnum(str, Enum):
```

Comment:
Like the comment says, we can remove this in favor of the official StrEnum class once Python 3.11 is our minimum supported version 👍